jobs/pgbackup/spec

```diff
@@ -62,17 +59,40 @@ properties:
     default: "*"
 
   pgbackup.s3.bucket:
-    description: "Amazon S3 bucket to store backups in"
-    default: "pgbackup"
+    description: "S3 bucket to store backups in"
   pgbackup.s3.path:
     description: "Path to prepend to backup filenames when storing in Amazon S3"
-    default: ""
+    default: "\"
   pgbackup.s3.access_key_id:
     description: "Access Key ID for storing backups in Amazon S3"
-    default: ""
   pgbackup.s3.secret_access_key:
     description: "Secret Access key for storing backups in Amazon S3"
-    default: ""
   pgbackup.s3.endpoint:
     description: "endpoint for backup/restore"
-    default: "s3.amazonaws.com"
+
+
+ pgbackup.s3.proxy.http.host:
+    description: "Hostname or Ip address of the proxy server, http://your_proxy_server"
+    default: ""
```

Comment:
Do we have use cases for separating http and https proxies ? If not consider a single set of properties for configuring both http and https